### PR TITLE
Fixes #11755: Document windows management usage

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/11755

### CODE REVIEW

File: 95_windows/50_technique_editor.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+[[technique-editor-dsc]]
+=== Technique editor with dsc
```
Comment (amousset) [inline 95_windows/50_technique_editor.txt]: DSC

**Comment by ncharles** [comment]
Commit modified

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**